tool/release_helper.sh

```diff
@@ -59,4 +59,6 @@ echo
 
 echo "Your Draft release PR can be found at: $PR_URL"
 echo
-echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
\ No newline at end of file
+echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
+
+exit
```

Comment:
How about adding new line to the end of file to remove this red sign?

Reply:
<img width="207" alt="Screenshot 2023-09-05 at 8 08 38 AM" src="https://github.com/flutter/devtools/assets/12115586/475177e4-8f6a-4d6c-9578-2b0943a0a229">